tests/filters.rs

```diff
@@ -0,0 +1,88 @@
+//! Tests for --exclude --include flags for suckit
+
+mod fixtures;
+
+use std::process::Command;
+use std::fs::read_dir;
+use std::process::Stdio;
+use fixtures::{get_file_count};
+use std::sync::Once;
+
+const SUCKIT: &'static str = "target/debug/suckit";
+const ADDR: &'static str = "http://0.0.0.0:8000";
+
+static START: Once = Once::new();
+
+#[test]
+fn test_include_exclude() {
+    START.call_once(|| {
+        fixtures::spawn_local_http_server();
+    });
+
+    include_filter();
+    include_multiple_filters();
+    exclude_filter();
+}
+
+fn include_filter() {
+    let output_dir = "w2";
+    let mut cmd = Command::new(SUCKIT)
+    .args(&[ADDR, 
+            "-o", "w2",
+            "-i", "mp[3-4]",
+            "-j", "16"])
+    .stdout(Stdio::inherit())
+    .stderr(Stdio::inherit())
+    .spawn()
+    .unwrap();
+    
+    let status = cmd.wait().unwrap();
+    assert!(status.success());
+    let paths = read_dir(output_dir).unwrap();
+    assert_eq!(paths.count(), get_file_count("*_mp3", output_dir));
```

Comment:
Because, we don't save things with extension now, we will need to change `_mp3` in future, once saving files with extensions is supported.

Reply:
#79